pkg/porter/generateManifest_test.go

```diff
@@ -1,32 +1,37 @@
 package porter
 
 import (
+	"context"
+	"errors"
 	"path/filepath"
 	"testing"
 
 	"get.porter.sh/porter/pkg/build"
+	cnabtooci "get.porter.sh/porter/pkg/cnab/cnab-to-oci"
 	"get.porter.sh/porter/pkg/config"
 	"get.porter.sh/porter/pkg/test"
+	"github.com/docker/docker/api/types"
 	"github.com/stretchr/testify/require"
 )
 
 func Test_generateInternalManifest(t *testing.T) {
 	testcases := []struct {
 		name         string
 		opts         BuildOptions
+		wantErr      string
 		wantManifest string
 	}{{
 		name:         "no opts",
 		opts:         BuildOptions{},
-		wantManifest: "original.yaml",
+		wantManifest: "expected-result.yaml",
```

Comment:
I believe the `archive_test.go` also helps to validate my changes since I removed the `digest` field from the yaml file. I also added explicit unit test for the logic in the callback function that handles tag. Let me know what do you think